src/rgen/Microsoft.Macios.Generator/Emitters/EnumEmitter.cs

```diff
@@ -95,6 +95,17 @@ void EmitExtensionMethods (TabbedWriter<StringWriter> classBlock, in Binding bin
 			getValueBlock.WriteLine (
 				"throw new NotSupportedException ($\"The constant {constant} has no associated enum value on this platform.\");");
 		}
+		classBlock.WriteLine ();
+
+		// get value from a handle, this is a helper method used in the BindAs bindings.
+		using (var getValueFromHandle =
+			   classBlock.CreateBlock ($"public static {binding.Name} GetValueFromHandle (NativeHandle handle)",
```

Comment:
Can this be internal instead of public? That way we don't have to worry about API diffs, etc.

Reply:
Can it when we are using it for bindings? I don't think so right?

Reply:
That depends on where these methods are going to be used. Is it always going to be from the same assembly?

Reply:
I think we can just use `GetValue`, it'll be an overload of the other `GetValue method:

```suggestion
			   classBlock.CreateBlock ($"public static {binding.Name} GetValue (NativeHandle handle)",
```

Reply:
Let me land this and change it in bgen too. Because bgen is already doing GetValueFromHandle. Sounds good?